mlos_bench/mlos_bench/tunables/tunable.py

```diff
@@ -63,6 +64,7 @@ def __init__(self, name: str, config: TunableDict):
         self._description = config.get("description")
         self._default = config["default"]
         self._values = config.get("values")
+        self._meta: Dict[str, Any] = config.get("meta", {})
```

Comment:
```suggestion
        self._meta: Dict[str, TunableValue] = config.get("meta", {})
```

Do you forsee a case where it should be more than this?

Reply:
actually, yes - e.g., something along the line of:
```json
"multiplier_suffix": {
    1024: "K",
    1048576: "M",
    1073741824: "G"
}
```
or, say,
```json
"constraints": {
    "less_than": ["innodb_io_capacity_max"]
}
```
etc.